go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/cpanato/action-mattermost-notify
 
-go 1.12
+go 1.13
+
+require (
+	github.com/mattermost/mattermost-server/v5 v5.3.2-0.20200114132950-e77ef9733c97
+	github.com/pkg/errors v0.9.0
+	go.uber.org/zap v1.13.0 // indirect
+	golang.org/x/crypto v0.0.0-20200109152110-61a87790db17 // indirect
+	gopkg.in/yaml.v2 v2.2.7 // indirect
```

Comment:
How did you generate the file? Are the indirect imports needed?

Reply:
i can build it again